src/gui/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { merge } from 'glamor'
+
+import TabsHeader from './tabsHeader'
+import TabsAction from './tabsActions'
+import styles from './tabs.styles'
+
+class Tabs extends Component {
+  state = {
+    selected: this.props.defaultKey,
```

Comment:
I know that you don't like inner state 😇

Otherwise, I give the `index` in that case, because `title` is a `PropTypes.node` (for example if I want to have an icon + label in my tab header). So `title`can't be a `key`, maybe I should add a `Tab#tabKey` prop.